j2cl-tasks/src/main/java/com/vertispan/j2cl/build/provided/J2clTask.java

```diff
@@ -49,12 +52,80 @@ public Task resolve(Project project, Config config) {
                 .map(input -> input.filter(JAVA_BYTECODE))
                 .collect(Collectors.toList());
 
+        Input ownStrippedBytecode = null;
```

Comment:
Need to remove this commit entirely, it will be in a follow-up, as it is more complex